utility/check.go

```diff
@@ -133,6 +155,12 @@ func CheckAvailability(resource string, regionSet string) (bool, string, error)
 		}
 	}
 
+	if resource == "db" {
+		if defaultRegion.Features.DBaaS && !defaultRegion.OutOfCapacity {
```

Comment:
I think the field got deleted somehow when you moved files? Can you take a look?